testsuite/tests/singlecluster/authorino/collisions/test_collisions.py

```diff
@@ -0,0 +1,62 @@
+"""Test auth policies collisions when attached to the same target"""
+
+import pytest
+
+from testsuite.httpx.auth import HeaderApiKeyAuth
+from testsuite.kuadrant.policy import has_condition
+from testsuite.kuadrant.policy.authorization.auth_policy import AuthPolicy
+
+pytestmark = [pytest.mark.authorino]
+
+
+@pytest.fixture(scope="module")
+def api_key(create_api_key, module_label):
+    """Creates API key Secret."""
+    return create_api_key("api-key", module_label, "api_key_value")
+
+
+@pytest.fixture(scope="module")
+def target(request):
+    """Returns the test target(gateway or route)"""
+    return request.getfixturevalue(request.param)
+
+
+@pytest.fixture(scope="module")
+def authorization(
+    request, gateway, route, cluster, blame, target, label, oidc_provider
+):  # pylint: disable=unused-argument
+    """Create an AuthPolicy with either gateway or route as target reference"""
+    auth = AuthPolicy.create_instance(cluster, blame("fp"), target, labels={"testRun": label})
```

Comment:
I forgot this before merging. This is on me...
The standalone tests are not passing.
You cant create AuthPolicy here as Authorino knows only AuthConfig.

Reply:
Quick fix for this would be to add `kuadrant_only` in marks or use `authorization` fixture already provided - which handles the changing from AuthPolicy to AuthConfig depending on if the tests are run as standalone or no.